Lib/sre_parse.py

```diff
@@ -11,7 +11,7 @@
 """Internal support module for sre"""
 
 # XXX: show string offset and offending character for all errors
-
+import collections.abc
```

Comment:
This may hurt startup performance, since collections.abc is a heavy module and re currently does not depend on it.